src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/GenericWhitespaceCheckTest.java

```diff
@@ -164,6 +173,29 @@ public void testGenericWhitespaceWithEmoji() throws Exception {
                 getPath("InputGenericWhitespaceWithEmoji.java"), expected);
     }
 
+    @Test
+    public void testClearState() throws Exception {
```

Comment:
Please add a comment here explaining why we need synthetic test.

Reply:
Done.

Reply:
I am ok, we can be fanatic on elimination of such test in next phase to let us focus on pitest now.